libqtile/widget/spacer.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, length=bar.STRETCH, width=None):
         base._Widget.__init__(self, length)
 
     def draw(self):
-        self.drawer.clear(self.bar.background)
+        self.drawer.clear(self.background)
```

Comment:
Shouldn't we fallback on `self.bar.background` if `self.background` isn't set?

```suggestion
        self.drawer.clear(self.background or self.bar.background)
```